Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -1809,6 +1838,107 @@ void HTMLElement::queue_a_popover_toggle_event_task(String old_state, String new
     };
 }
 
+// https://html.spec.whatwg.org/multipage/popover.html#light-dismiss-open-popovers
+void HTMLElement::light_dismiss_open_popovers(UIEvents::PointerEvent const& event, const GC::Ptr<DOM::Node> target)
```

Comment:
Is there a reason we need to pass in the target instead of using `event.target()` in step 2?

(Also nit: east const)

Reply:
I tried that initially but found that `event.target()` was always null (perhaps it is set after propagation?)

Reply:
Yeah we don't seem to implement any of the "initialize a FooEvent" algorithms from that spec.

Can you add some FIXMEs to explain why `target` is a parameter?

Reply:
done :+1: